lib/services/local/events.go

```diff
@@ -1637,7 +1637,7 @@ func (p *headlessAuthenticationParser) parse(event backend.Event) (types.Resourc
 
 func newAccessListParser() *accessListParser {
 	return &accessListParser{
-		baseParser: newBaseParser(backend.Key(accessListPrefix)),
+		baseParser: newBaseParser(backend.Key(accessListPrefix, "")),
```

Comment:
```suggestion
		baseParser: newBaseParser(backend.ExactKey(accessListPrefix)),
```

Can we do the same change to `newAccessListMemberParser`? Is this a widespread problem?

Reply:
Will do.

This happens elsewhere, and the pattern I've used here is used elsewhere. However, this is one of the few keys with overlap, so we've lucked into not hitting this before.

I'm wondering if the watcher should error on key overlap rather than silently failing like this.